docs/3.0/customization.md

```diff
@@ -567,3 +567,19 @@ end
 
 <Image src="/assets/img/3_0/customization/click-row-to-view-record.gif" width="" height="" alt="Click to view record in Avo" />
 </Option>
+
+## Associations lookup list limit
+
+<VersionReq version="3.14.0" />
```

Comment:
`3.14.0` is the current release version and `associations_lookup_list_limit`is not included in it.

We do a minor release on the first Tuesday of each month so `3.15.0` will include this feature. However, we’ll soon issue a patch release with some fixes, so it will be available starting from  `3.14.1`

```suggestion
<VersionReq version="3.14.1" />
```